### ISSUE TITLE
:required option should generate an 'aria-required' attribute

### ISSUE THREAD

**Issue by ckundo** [heartcombo/simple_form#780]
The WAI-ARIA specification provides attributes that help assistive technologies render content more coherently to persons with disabilities. One of those attributes is aria-required that when set to true, will indicate to a screen reader that it should announce that the field is required. See http://www.w3.org/TR/wai-aria-practices/#ariaform

I propose adding behavior to simple form's builder that will generate the aria attribute automatically when the 'required' option is used. Are there any immediate objections? Otherwise I'd like to take this on and submit a pull request.
